gossip/src/crds_gossip_pull.rs

```diff
@@ -1467,7 +1467,7 @@ pub(crate) mod tests {
         let keypair = Keypair::new();
         let node = {
             let mut node =
-                ContactInfo::new_localhost(&keypair.pubkey(), /*wallclock:*/ rng.gen());
+                ContactInfo::new_localhost(&keypair.pubkey(), /*wallclock:*/ timestamp());
```

Comment:
Why change this from RNG to timestamp?

Reply:
`get_max_bloom_filter_bytes` is a step function. So constant for a little bit between where it changes.
`set_tpu_vote` was also missing for `ContactInfo::new_localhost`.
`ContactInfo` uses varint encoding for `wallclock` serialization.

Combination of these 3 was making the test flaky because the serialized size of contact-info was not fixed and could cross the step function boundaries.
Changing wallclock from `rng.gen()` to `timestamp()` was the smallest change for the purposes of this PR to address flakiness without changing the test coverage much.